Fix rebuilding

skisela opened the pull request:
Fixes #468

nforro commented on rebasehelper/application.py:
```diff
@@ -804,6 +779,26 @@ def apply_changes(self):
             logger.warning('changes.patch was not applied properly. Please review changes manually.'
                            '\nThe error message is: %s', six.text_type(e))
 
+    def fix_specfile(self, results_dir):
```
This shouldn't be called `fix_specfile`, that's misleading. Something like `prepare_next_run` would be more fitting.

nforro commented on rebasehelper/application.py:
```diff
@@ -846,18 +841,22 @@ def run(self):
+                        if self.conf.builds_nowait and not self.conf.build_tasks:
+                            return
+                        self.run_package_checkers(self.results_dir, category='RPM')
+                        break
```
Just a nitpick, but I think this `break` fits more into a `else` block of the `try ... except` statement.

skisela commented:
Fixed, thanks!

I also added commit https://github.com/rebase-helper/rebase-helper/pull/499/commits/5c57d19a95f3c6bfb2f8a5ace41ed19952ced135 so that `setup.py` knows about the freshly added `helpers` package created by @FrNecas .
